src/crypto/wolfssl.c

```diff
@@ -271,35 +258,10 @@ void us_internal_free_loop_ssl_data(struct us_loop_t *loop) {
     }
 }
 
-// we ignore reading data for ssl sockets that are
-// in init state, if our so called budget for doing
-// so won't allow it. here we actually use
+// we throttle reading data for ssl sockets that are in init state. here we actually use
 // the kernel buffering to our advantage
-int ssl_ignore_data(struct us_internal_ssl_socket_t *s) {
-
-    // fast path just checks for init
-    if (!SSL_in_init(s->ssl)) {
-        return 0;
-    }
-
-    // this path is run for all ssl sockets that are in init and just got data event from polling
-
-    struct us_loop_t *loop = s->s.context->loop;
-    struct loop_ssl_data *loop_ssl_data = loop->data.ssl_data;
-
-    // reset handshake budget if new iteration
-    if (loop_ssl_data->last_iteration_nr != us_loop_iteration_number(loop)) {
-        loop_ssl_data->last_iteration_nr = us_loop_iteration_number(loop);
-        loop_ssl_data->handshake_budget = MAX_HANDSHAKES_PER_LOOP_ITERATION;
-    }
-
-    if (loop_ssl_data->handshake_budget) {
-        loop_ssl_data->handshake_budget--;
-        return 0;
-    }
-
-    // ignore this data event
-    return 1;
+int ssl_is_low_prio(struct us_internal_ssl_socket_t *s) {
+    return SSL_in_init(s->ssl);
```

Comment:
This is not the same as openssl.c

Reply:
Yes, please read my comment from 27 Apr, there I explained it: https://github.com/uNetworking/uSockets/pull/97#issuecomment-620092265